builder/vsphere/common/storage_config.go

```diff
@@ -65,49 +86,32 @@ import (
 //	    "disk_controller_index": 1
 //	  }
 //	],
-//
 // ```
 //
-// In HCL2:
-// ```hcl
-//
-//	disk_controller_type = ["pvscsi", "pvscsi"]
-//	storage {
-//	   disk_size = 15000,
-//	   disk_controller_index = 0
-//	}
-//	storage {
-//	   disk_size = 15000
-//	   disk_controller_index = 0
-//	}
-//	storage {
-//	   disk_size = 15000
-//	   disk_controller_index = 1
-//	}
-//	storage {
-//	   disk_size = 15000
-//	   disk_controller_index = 1
-//	}
-//
-// ```
+
 type DiskConfig struct {
-	// The size of the disk in MiB.
+	// Specifics the size of the disk in MiB.
```

Comment:
```suggestion
	// Specifies the size of the disk in MiB.
```

Doesn't that change go against a previous effort to remove instances of `specifies ...`?

Reply:
Addressed in 42c68d4.